_includes/pbc/all/install-features/202311.0/install-the-service-points-feature.md

```diff
@@ -1535,8 +1624,8 @@ class ShopApplicationDependencyProvider extends SprykerShopApplicationDependency
 
 Make sure the following widgets have been registered by adding the respective code snippets to a Twig template:
 
-| WIDGET                   | VERIFICATION                                                                                                                                                 |
-|--------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| WIDGET                   | VERIFICATION                              |
+|--------------------------|-----------------------------------------------------------------|
 | ServicePointSearchWidget | `{% raw %}{%{% endraw %} widget 'ServicePointSearchWidget' args [...] only {% raw %}%}{% endraw %}{% raw %}{%{% endraw %} endwidget {% raw %}%}{% endraw %}` |
```

Comment:
What do I need to check to verify it's working?

Reply:
As a developer, you should add it somewhere on the twig page (for example checkout address molecule). But I didn't see previous examples where we provide such verifications for widgets.

Reply:
Yeah, but it doesn't have to be very detailed. All we are missing is a criteria to check against. After adding the widget, what should I see when I open the page with it?

Reply:
make sense, could you please add to the verification block: "Ensure that you can find and select a service point for your products on the `http://www.mysprykershop.com/en/checkout/address` page."